Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs

```diff
@@ -803,24 +825,29 @@ private string CreateProjectFile(string testSpecificProjectXml, string sonarQube
 
         #region Assertions methods
 
-        private string CheckProjectSpecificOutputStructure(string rootOutputFolder)
-        {
-            Directory.Exists(rootOutputFolder).Should().BeTrue("Expected root output folder does not exist");
+        private string CheckProjectSpecificOutputStructure(string rootOutputFolder) =>
+            CheckProjectSpecificDirectoryStructure(rootOutputFolder, "output");
 
-            // We've only built one project, so we only expect one directory under the root output
-            Directory.EnumerateDirectories(rootOutputFolder).Should().HaveCount(1, "Only expecting one child directory to exist under the root analysis output folder");
+        private string CheckProjectSpecificConfigStructure(string rootConfigFolder) =>
+            CheckProjectSpecificDirectoryStructure(rootConfigFolder, "config");
+
+        private string CheckProjectSpecificDirectoryStructure(string rootFolder, string logType)
+        {
+            Directory.Exists(rootFolder).Should().BeTrue($"Expected root {logType} folder does not exist");
 
-            var fileCount = Directory.GetFiles(rootOutputFolder, "*.*", SearchOption.TopDirectoryOnly).Count();
-            fileCount.Should().Be(0, "Not expecting the top-level output folder to contain any files");
```

Comment:
[education] where did this go?

Reply:
See line 842. GitHub and VS don't do very good job at showing compared files.
Old method: 
`CheckProjectSpecificOutputStructure`

New methods: 
`CheckProjectSpecificOutputStructure` - old method change to a proxy/entry point
`CheckProjectSpecificConfigStructure` - sibling
`CheckProjectSpecificDirectoryStructure` - parametrized version of the original `CheckProjectSpecificOutputStructure`